src/components/DatabaseList/DatabaseList.module.css

```diff
@@ -0,0 +1,112 @@
+.MainPage {
+  /* Divided into two rows */
+  margin: 0;
+  background: #f1f1f1;
+  height: 100vh;
+  display: grid;
+  grid-template-rows: 80px 4fr;
+  overflow-x: hidden;
+}
+.TopBarSection {
+  background-color: black;
+}
+.MainSection {
+  /* divided into two columns */
+  background-color: #f1f1f1;
+  display: grid;
+  grid-template-columns: 1.3fr 6fr;
+  overflow: hidden;
+}
+
+.SideBarSection {
+  overflow: hidden;
+  overflow-y: scroll;
+}
+
+.SideBarSection::-webkit-scrollbar {
+  display: none;
+  min-width: fit-content;
+}
+
+.SideBarSection {
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
+
+.MainContentSection {
+  /* divided into two rows */
+  display: grid;
+  grid-template-rows: 80px 4fr;
+  overflow: hidden;
+}
+
+.ContentSection {
+  display: flex;
+  flex-direction: column;
+  overflow-x: hidden;
+  overflow-y: auto;
+  padding: 2rem;
+}
+.ContentSection::-webkit-scrollbar {
+  width: 10px;
+  height: 0px;
+}
+.ContentSection::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+.ContentSection::-webkit-scrollbar-thumb {
+  border-radius: 25px;
+  background: #008ac1;
+}
+.ContentSection::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
+
+.DatabaseTableHead {
+  text-transform: uppercase;
+  padding: 1rem;
+  color: #008ac1;
+  text-align: center;
+  grid-row: 1;
+}
+
+.DatabaseTableRow {
+  display: grid;
+  grid-auto-columns: 1fr;
+  color: #000000;
+  grid-template-columns: (5, minmax(0, 1fr));
```

Comment:
<img width="358" alt="Screenshot 2021-05-20 at 16 44 49" src="https://user-images.githubusercontent.com/24539185/118989602-d54ab300-b98a-11eb-821e-24a11a6bf69d.png">
let correct this as well: `grid-template-columns: repeat(5, minmax(0, 1fr));`
i believe u left out the `repeat` function

Reply:
oops I forgot.